### PR TITLE
test(block): Add screenshot test for long Block and Block Section props

### PR DESCRIPTION
**Pull request by macandcheese**
**Related Issue:** #3280

### CODE REVIEW

**Comment by macandcheese** [comment]
Chromatic link: https://www.chromatic.com/test?appId=6266d45509d7eb004aa254fb&id=653a97a872bb1a8cc020f92f

File: packages/calcite-components/src/components/block/block.stories.ts [hunk 1]
```diff
@@ -235,6 +235,40 @@ export const loadingWithNoStatusNorSlottedIcon_TestOnly = (): string =>
     </calcite-block>
   `;
 
+export const longWrappingTextInBlockAndBlockSection_TestOnly = (): string =>
+  html`
+    <calcite-panel style="width:250px">
+      <calcite-block
+        collapsible
+        open
+        heading="Planes, trains, and automobiles are some examples of modes of transportation"
```
Comment (driskull) [inline packages/calcite-components/src/components/block/block.stories.ts:244]: Might be nice in the future if we had a JSON file with sample content like this so all of our storybook screenshot tests could be consistent with good content. Maybe JSON file with `paragraph`, `sentence`, `snippet`, `heading`, etc. Could be something as a "good first issue" to cleanup.

**Review by driskull — APPROVED** [review]
👍

File: packages/calcite-components/src/components/block/block.stories.ts (hunk 1 above)
Comment (macandcheese) [reply packages/calcite-components/src/components/block/block.stories.ts:244]: Agreed `sampleText.long`, `sampleText.short` would be nice helpers to be able to reference. 

Or maybe a helper to generate "esri ipsum" with geography terms for a given character count... `getSampleText(400)`

**Pull request merged**